docs/using-controller-manager-with-kubeadm.md

```diff
@@ -52,8 +52,13 @@
 
 - Create a configmap containing the cloud configuration for cloud-controller-manager.
 
+   Encode your `cloud.conf` file content using base64: `base64 -w 0 cloud.conf`
```

Comment:
Normally, the file is located in `/etc/kubernetes/cloud-config`, so please use `/etc/kubernetes/cloud-config` explicitly.

Reply:
@lingxiankong Thanks for pointing it, 
@johscheuer  PR recently merged for consistency of cloud-config around docs
https://github.com/kubernetes/cloud-provider-openstack/pull/429/files, In my suggestion we can follow the same approach @lingxiankong what you think?

Reply:
yeah, I agree, it's important to keep the wording consistent across the repo :-)